Remove duplicate 12.5 warning

tas50 opened the pull request:
Custom resource themselves are entirely new for 12.5. There's no need to call out this one small change between LWRPs and custom resources.

jamescott commented:
https://github.com/chef/chef-docs/commit/6bf8a9d6c6083365562d9265f9f8bdf75256090a

sean-horn commented:
closing in favor of https://github.com/chef/chef-docs/commit/6bf8a9d6c6083365562d9265f9f8bdf75256090a

tas50 committed:
Remove duplicate 12.5 warning

Custom resource themselves are entirely new for 12.5. There's no need to call out this one small change between LWRPs and custom resources.

davidwrede commented:
![gif-keyboard-14488549827822945977](https://cloud.githubusercontent.com/assets/1634752/24377913/e44222de-12f5-11e7-869c-adc44bf208b7.gif)
